config/locales/nl.yml

```diff
@@ -84,6 +84,7 @@ nl:
       user: Gebruiker
       category: Categorie
       billed: Gefactureerd
+      unbill: Herzie facturatie
```

Comment:
changed to 'facturatie terugdraaien', it does stick out a little at the the end of the line but I think this is better then 'terugdraaien'